jobs/generation/Utilities.groovy

```diff
@@ -637,7 +637,9 @@ class Utilities {
             '\'type_traits\' file not found', // This is here for certain flavors of clang on Ubuntu, which can exhibit odd errors.
             '\'typeinfo\' file not found', // This is here for certain flavors of clang on Ubuntu, which can exhibit odd errors.
             'Only AMD64 and I386 are supported', // Appears to be a flaky CMAKE failure
-            'java.util.concurrent.ExecutionException: Invalid object ID'
+            'java.util.concurrent.ExecutionException: Invalid object ID',
+            '.*XmlException.', // This is here until NuGet cache corruption issue is root caused and fixed.
```

Comment:
I'd avoid this particular regex.  The second one seems to catch all of the cases we've seen lately.  I agree this one is a bit broad and would catch a lot of extra cases.